src/Microsoft.Extensions.Configuration.AzureAppConfiguration/IConfigurationRefresher.cs

```diff
@@ -42,14 +42,6 @@ public interface IConfigurationRefresher
         /// <param name="cancellationToken">The cancellation token to cancel the operation.</param>
         Task<bool> TryRefreshAsync(CancellationToken cancellationToken = default);
 
-        /// <summary>
-        /// Sets the cached value for key-values registered for refresh as dirty.
-        /// A random delay is added before the cached value is marked as dirty to reduce potential throttling in case multiple instances refresh at the same time.
-        /// </summary>
-        /// <param name="maxDelay">Maximum delay before the cached value is marked as dirty. Default value is 30 seconds.</param>
-        [Obsolete("SetDirty will be deprecated in a future release. Please use the 'ProcessPushNotification' method instead for push-model based configuration refresh.")]
```

Comment:
We dont need the implementation in `AzureAppConfigurationRefresher` anymore. Consequently, the SetDirty method in `AzureAppConfigurationProvider` can be a private method.